server/src/sharepoint/sharepoint.service.ts

```diff
@@ -6,6 +6,21 @@ import {
 import * as Request from 'request';
 import { ConfigService } from '../config/config.service';
 
+// {
+//  Files: []
+//  Folders: []
+// }
+
+function unnest(folders) {
```

Comment:
Can you clarify and document the structure of the input and outputs from this function?
This is my _guess_ from reading the code, but just want to double check with you:
```
/**
 * @param { 
       { 
          Files: [],
          Folders: [],
       }[]
     } folders - array of "fileFolder" objects. "fileFolder" objects have a list of files and a recursive list of fileFolder objects.

 * returns {Document Objects []}
 */
 ```